introduction/week-2/lesson.md

```diff
@@ -32,107 +32,148 @@ We will now do some pair programming.
 
 ## Sorting algorithm game
 
-This is a team exercise.  It works well for any number of teams of 3-5 students.
-
+This is a team exercise. It works well for any number of teams of 3-5 students.
 
 There is a longer planning phase which all teams do at once in their teams and then a short ‘attempt’ phase where each team in sequence can attempt to run their prepared plan on a set of volunteers.
 The challenge for students
 Your team’s goal is to organise a row of ‘people’ as quickly as possible while following all the rules given.
 
-
 Your group (ideally 3-5 students) will first work together to define the strategy and create a set of instructions needed to complete the task.
 
-
 When it is time to test your planned instructions, your group will split into planners and one executor:
-* The ‘planners’ communicate one instruction at a time to the executor, and receive responses from them
-* The ‘executor’ will execute the instructions and carry out the operation, according to strict rules, but will not take part in decision-making during the test.
-
-
-  
-
 
+- The ‘planners’ communicate one instruction at a time to the executor, and receive responses from them
+- The ‘executor’ will execute the instructions and carry out the operation, according to strict rules, but will not take part in decision-making during the test.
 
 The group of ‘people’ to be ordered will be assigned one value each at random, written on paper, visible to your team’s executor, but hidden to your team’s planners.
 
-
 The group of people will be placed one each on numbered slots, so that your planners can refer to them by position.
 
-
 There will be one extra empty slot called ‘waiting-room’ (‘WR’ in the diagram).
 
-
 Your team can decide whether you will order the people in ascending or descending order.
 The Executor’s role
-* One of your team will take the role of executor during the test of your instructions
-* The executor will receive a command and the input needed to carry out the command (e.g. “COMPARE: is value in slot 3 is larger than value in slot 7?”)
```

Comment:
Sorry for these! Just realised only after I committed that Prettier was running in the background and changed them.

Happy to revert them if you think it's worth it?